python/rmm/allocators/torch.py

```diff
@@ -11,19 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+rmm_torch_allocator = None
+
 try:
     from torch.cuda.memory import CUDAPluggableAllocator
 except ImportError:
-    rmm_torch_allocator = None
+    pass
 else:
-    import pathlib
+    from torch.cuda import is_available
+
+    if is_available():
+        import pathlib
+
+        # To support editable installs, we cannot search for the compiled torch
+        # allocator so relative to the current file because the current files
```

Comment:
nit: Clarification and typo.

```suggestion
        # allocator .so relative to the current file because the current file
```